### PR TITLE
LPS-164586 baseline

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://issues.liferay.com/browse/LPS-164586

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -1073,7 +1073,8 @@ public ObjectEntry updateObjectEntry(
 		_startWorkflowInstance(userId, objectEntry, serviceContext);
 
 		_deleteFileEntries(
-			values, objectEntry.getObjectDefinitionId(), transientValues);
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:1076]: @marcelabc why do we need to pass the objectEntry.getValues() now?

**Review by gabrielwas — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 1 above)
Comment (marcelabc) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java]: I fixed, thanks!

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:forward

**Comment by marcelabc** [comment]
The failures are flaky.

Sent directly to Brian: https://github.com/brianchandotcom/liferay-portal/pull/124209

**Pull request closed without merging**